### PR TITLE
new year, new design (system)

### PR DESCRIPTION
**Pull request by tlwr**
TLDR
----

#### Demo

Use https://tlwr.github.io/fourth-wall

#### Screenshot

image

#### What

Redesigns fourth wall to use the gov.uk design system and "cards".

The design language of the colours was unclear and it was unreadable on large
screens.

Cards are used instead of rows.

The colours have been moved to a status-stripe on the top of each card.

Purpose
-------

The purpose of this redesign is to:

- Make the design/colours less loud; when put on a build monitor next to an
alerting/monitoring dashboard a more muted design will not distract from the
more important monitor.

- Better information density; when put on a display in a very
wide configuration it was difficult to read the pull requests.

Not in scope
------------

- Actually fixing the colours - someone who can see them properly should have a
go at this

- Fixing the structure of this project. I've been meaning to move to using
Vue.js 2 or something similar so we can do template rendering in a better way,
however that should come in a separate PR.

- Fixing the tests, phantomjs problems on travis have haunted this repo for a while, when restructuring this repo / using a different JS framework, we can test this project better.

Screenshots
-----------

### Thin displays (single column)

Information density is objectively worse

image

compared to

image

### Wide displays (cards are shown)

In my opinion it is much easier to read this way

image

compared to

image

### Typical (in my experience) display

image

compared to

image

#### Other

#72 was an accidental force push to the wrong repo... 🤦‍♂️

### CODE REVIEW

**Comment by issyl0** [comment]
If you rebase on `gh-pages`, we can merge this. 🚀

**Comment by aliuk2012** [comment]
As this is not on GOV.UK Domain, you shouldn't be using the transport font or the crown crest. So you should be able to exclude the font fonts in this PR and it will default to Arial.

Also I think this PR needs some tests for the new methods in master-view.js and pull-view.js

**Comment by aliuk2012** [comment]
Can I suggest also updating the image used in the README.md?
